apps/ui/lib/ui_web/live/gaining_losing_trades_live.ex

```diff
@@ -0,0 +1,120 @@
+defmodule UiWeb.GainingLosingTradesLive do
+  use Phoenix.LiveView
+  alias Timex, as: T
+
+  def render(assigns) do
+    ~L"""
+      <div class="box box-success">
+        <div class="box-header with-border">
+          <h3 class="box-title">Gaining / Losing - <%= @symbol %></h3>
+
+          <div class="box-tools pull-right">
+            <button type="button" class="btn btn-box-tool" data-widget="collapse"><i class="fa fa-minus"></i>
+            </button>
+            <button type="button" class="btn btn-box-tool" data-widget="remove"><i class="fa fa-times"></i></button>
+          </div>
+        </div>
+        <!-- /.box-header -->
+        <div class="box-body" style="">
+          <div class="row">
+            <div class="col-md-8">
+              <%= if length(@data.symbols) > 0 do %>
+                <div class="col-xs-5">
+                  <form phx-change="change-symbol" id="change-symbol">
+                    <select name="selected_symbol" class="form-control col-xs-3">
+                      <%= for row <- @data.symbols do %>
+                          <option value="<%= row %>"
+                          <%= if row == @symbol do %>
+                            selected
+                          <% end %>
+                          ><%= row %></option>
+                        <% end %>
+                    </select>
+                  </form>
+                </div>
+              <% end %>
+              <div class="chart-responsive">
+                <canvas id="doughnutChart" height="160" width="329" style="width: 329px; height: 160px;"></canvas>
+                  <script id="chart-<%= Base.encode64(:erlang.md5(@symbol)) %>">
+                    renderDoughnutChart([<%= for l <- get_chart_data(@data.chart_data.rows, @symbol) do %>"<%= l %>",<% end %>])
+                  </script>
+              </div>
+              <!-- ./chart-responsive -->
+            </div>
+            <!-- /.col -->
+            <div class="col-md-4">
+              <ul class="chart-legend clearfix">
+                <li><i class="fa fa-circle-o text-green"></i> Gaining</li>
+                <li><i class="fa fa-circle-o text-red"></i> Losing</li>
+              </ul>
+            </div>
+            <!-- /.col -->
+          </div>
+          <!-- /.row -->
+        </div>
+        <!-- /.box-body -->
+        <div class="box-footer no-padding" style="">
+        </div>
+        <!-- /.footer -->
+      </div>
+    """
+  end
+
+  def mount(%{}, socket) do
+    [from, to] = Hefty.Utils.Datetime.get_timestamps(T.today())
+
+    symbols =
+      Hefty.Trades.fetch_trading_symbols(from, to)
```

Comment:
why only trading symbols? I should be able to select any symbol

btw. I would assume that fetching trading symbols would query `naive_trader_settings` for a list of `enabled`